test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,28 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity *nodeAffinity
+	InterPodAffinity *interPodAffinity
+}
+
+// interPodAffinity priority configuration details.
+type interPodAffinity struct {
```

Comment:
ah yeah, so youre not against having the NodeSelectorReq, you just want it to be carried in by a high level type.  totally agree there.